pom.xml

```diff
@@ -67,7 +67,7 @@
   <properties>
     <revision>2.6.3</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.308</jenkins.version>
+    <jenkins.version>2.289.1</jenkins.version>
```

Comment:
Does not match https://github.com/jenkinsci/credentials-plugin/blob/15eb87b73807f41a5c6fadf59521ba755262d538/pom.xml#L92

Reply:
Thanks, bumped to 2.303.1